### ISSUE TITLE
Projected + built blocks appears to exceed blocklimits. 

### ISSUE THREAD

**Issue by TAGur1t** [DraconisCluster/DX-Issues#42]
** tl; dr **
Projected # of refineries exceeds blocklimit values

** Full info **
Player report, Refinery blocks can be projected on a grid and then in certain cases exceeds with projections. e.g. 4 Refineries are projected and then 

"So I projected this refinery, it had 4 refineries by default. I was able to add like 10 more refineries afterwards, going over my limit. It showed up as 10/10 refinieries with the !blocklimit mylimit command despite there being 13. I wasn't getting any warnings either when this happened
Usually I get a warning saying I'm over a limit

**Screenshots**
--If applicable, add screenshots to help explain your problem.--
